client/require.js

```diff
@@ -0,0 +1,17 @@
+const moduleStore = {
```

Comment:
This is a custom require function which will used by the app's `main.js`.
This is used to load modules from the bundle.